api_spec.yaml

```diff
@@ -1266,17 +1472,285 @@ paths:
+                            type: object
+                            properties:
+                                tenantId:
+                                    type: string
+                                providerConfig:
+                                    $ref: "#/components/schemas/thirdPartyProvider"
+            responses:
+                200:
+                    description: Success
+                    content:
+                        application/json:
+                            schema:
+                                type: object
+                                properties:
+                                    status:
+                                        type: string
+                                        default: "OK"
+                                    createdNew:
+                                        type: boolean
+                                        default: true
+                401:
+                    description: Unauthorised access
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+                                enum:
+                                    - Unauthorised access
+        delete:
+            tags:
+                - Multitenancy
+            summary: Delete thirdParty config for a tenant
+            parameters:
+                - name: tenantId
+                  in: query
+                  required: true
+                  schema:
+                      type: string
+                      example: "tenant1"
+                - name: thirdPartyId
+                  in: query
+                  required: true
+                  schema:
+                      type: string
+                      example: "google"
+                - name: authorization
+                  in: header
+                  required: true
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
+            responses:
+                200:
+                    description: Success
+                    content:
+                        application/json:
+                            schema:
+                                type: object
+                                properties:
+                                    status:
+                                        type: string
+                                        default: "OK"
+                                    didConfigExist:
+                                        type: boolean
+                                        default: true
+                401:
+                    description: Unauthorised access
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+                                enum:
+                                    - Unauthorised access
+    /dashboard/api/multitenancy/core-config/list:
```

Comment:
core config list is unrelated to multitenancy, can we use a different path?

`/dashboard/api/core/config/list`

Reply:
Done